packages/babel-preset-env/README.md

```diff
@@ -18,22 +18,20 @@ Without any configuration options, @babel/preset-env behaves exactly the same as
 
 You can also configure it to only include the polyfills and transforms needed for the browsers you support. Compiling only what's needed can make your bundles smaller and your life easier.
 
-This example only includes the polyfills and code transforms needed for coverage of users > 0.25%, ignoring Internet Explorer 11 and Opera Mini. We use [browserslist](https://github.com/ai/browserslist) to parse this information, so you can use [any valid query format supported by browserslist](https://github.com/ai/browserslist#queries).
+By default, it includes the polyfills and code transforms needed for [this modern browsers](http://browserl.ist/?q=defaults).
```

Comment:
Actually, it's not by default for preset-env. [We are rewriting](https://github.com/babel/babel/blob/master/packages/babel-preset-env/src/targets-parser.js#L126) defaults to targets specified by the user. It's because w/o any configuration we want all transofrm/polyfills to be applied. It's safe and easier to migrate from ES2015 preset.

So by default, it includes the polyfills and code transform needed for **all browsers**.

Reply:
Is a having different behavior with Autoprefixer and other tools is a great idea for community and long-term?

Also Browserslist `defaults` has a lot of browsers. Most of the browsers which are really in use by someone. http://browserl.ist/?q=defaults

Note, that if you will take *all* browsers, it means Chrome 3, old Opera and many browsers which you never tested. Are you sure, that you truly support them?